apps/music/js/ui/tab_bar.js

```diff
@@ -15,25 +14,20 @@ var TabBar = {
     null
   ],
 
-  get view() {
-    return document.getElementById('tabs');
+  get views() {
+    return document.getElementsByClassName('tabs');
```

Comment:
Right now, it looks like you have a single `TabBar` global that handles *all* three tab bars at once. I think it would be better to turn `TabBar` into a class that you can instantiate and add as a member of `TilesView`, `ListView`, etc. For example, `TilesView.init()` would have something like this:

```
init: function() {
  this.tab_bar = new TabBar( this.view.querySelector('.tabs') );
  // ...
}
```

We'd have to figure out how to send messages to the right `TabBar` then. Maybe we could just use DOM events for now (and switch to threads.js when we're ready).